python/spacewalk/server/importlib/backendLib.py

```diff
@@ -518,7 +510,38 @@ def sanitizeValue(value, datatype):
     if isinstance(datatype, DBdate):
         return str(value)[:10]
     if isinstance(datatype, DBint):
-        return int(value)
+        try:
+            value = _sanitize_number_string(value)
+            return int(float(value))
+        except ValueError as e:
+            raise ValueError(f"Cannot convert {value} to int") from e
+    return value
+
+
+def _sanitize_number_string(value):
+    """
+    Enable number-like strings to be converted by
+    int() or float()
+    """
+    if not isinstance(value, str):
+        return value
+
+    # Allow "1 234"
+    value = value.replace(" ", "")
+    # 1,234 cannot be processed by float()
+    value = value.replace(",", ".")
```

Comment:
This one is dangerous, different countries assign different meaning to commas in numbers. `1,234` US is `1234`, but `1,234` in Germany is `1.234` US.

Reply:
Yes, but `1.234` is already handled by `float()` and `float()`, afaik, never accepts `1,234` regardless of the local settings?